.github/workflows/any-branch-uploads.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     - uses: actions/checkout@v3
     - name: Build spec
       run: |
-        docker run -v `pwd`/docs/v1:/usr/src/p4-spec p4lang/p4rt-asciidoc :latest make
+        docker run -v `pwd`/docs/v1:/usr/src/p4-spec p4lang/p4rt-asciidoc:latest make
```

Comment:
@Dscano I believe you need to build the dockerfile before you` docker run ...` in all the CI scripts, or it'll fail to find the image. This is similar to what you had to do in the original MR: https://github.com/p4lang/p4runtime/blob/7e981d865a2a2694d5ffb019b63a56fa40739bfb/.github/workflows/spec.yml#L28

It's a consequence of not having a public docker image. Unfortunately, this is not exposed until CI runs after the merge occurs. Can you make sure this is added to all the CI files accordingly? Thanks.

Reply:
done